pkg/daemon/ceph/osd/remove.go

```diff
@@ -83,17 +83,21 @@ func removeOSD(clusterdContext *clusterd.Context, clusterInfo *client.ClusterInf
 	if err != nil {
 		logger.Errorf("failed to exclude osd.%d out of the crush map. %v", osdID, err)
 	}
-
+	forceRemoval, exitIfNotSafe := forceRemovalCallback(osdID)
 	// Check we can remove the OSD
 	// Loop forever until the osd is safe-to-destroy
 	for {
 		isSafeToDestroy, err := client.OsdSafeToDestroy(clusterdContext, clusterInfo, osdID)
 		if err != nil {
 			// If we want to force remove the OSD and there was an error let's break outside of
 			// the loop and proceed with the OSD removal
-			if forceOSDRemoval {
+
+			if forceRemoval {
```

Comment:
It is strange that we need duplicate logic in this error condition compared to below on lines 113-123, but it's really a separate question from this PR. It's not clear how to refactor it now.

Reply:
I'll open a issue to track this but may be we can have helper function to handle the `forceRemoval` block